docs/qlin_genetic/filters_manager/filters_manager.md

```diff
@@ -0,0 +1,47 @@
+# Filter Manager
+
+The Filter Manager allows you to save, combine, and share one or more queries. A query is based on one or more filters from the filter sidebar.
+
+## Filters
+### Saving a Filter
+
+The floppy disk icon 💾 allows you to save a filter for later use.
+The dropdown menu **My Filters** allows you to load a previously saved filter.
+
+Note that saving a filter only preserves the filter rules and not the result. The result may change if, for example, the data is updated.
+
+### Naming a Filter
+
+The pencil icon ![pencil](editicon.png) allows you to edit the filter name to find it more easily later.
+
+### Sharing a Filter
+
+After [saving a filter](#saving-a-filter), the ![share](shareicon.png) icon allows you to copy its URL to share it with other Qlin Genetics users.
+
+### Duplicating a Filter
+
+The 📑 icon located to the right of the floppy disk icon 💾 allows you to create a copy of the filter, including the queries contained within it.
+
+### Deleting a Filter
+
+The 🗑️ icon deletes the filter as well as all the queries included within it.
+
+## Queries
+
+To create a first query, apply a filter using the filter sidebar.
+
+To create additional queries, use the **New Query** button or duplicate an existing query with the 📑 icon."
+
+### Current Query
+
+The current query is identified by a blue background color on its banner. The table data is filtered by the current query. Applying new filters adds to the current query.
+
+### Joins
+
+It is possible to modify the join of the criteria using the **And / Or** hyperlink.
+
+When selecting multiple queries using the checkboxes to the left of the query names (Q1, Q2, etc.), the **Combine** button allows you to create a new query with a join of the selected queries. The **…** button allows you to select the join type And / Or.
```

Comment:
And / Or ne sont pas en gras comme en français.
Appliquer la même logique de "gras" uniformément:
- Soit avec la barre oblique en gras
- Soit la barre oblique exclue du gras